projects/media-viewer/src/lib/annotations/annotations.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { Annotation } from './annotation.model';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'mv-annotations',
+  styleUrls: ['./annotations.component.scss'],
+  templateUrl: './annotations.component.html'
+})
+export class AnnotationsComponent {
+
+  @Input() annotations: Annotation[];
+  @Input() selectedAnnotation: Subject<string>;
```

Comment:
this is not needed anymore, as we're just listening to the focusOut event on the annotation itself